components/HeaderTop.vue

```diff
@@ -45,7 +45,8 @@ export default {
     }
   },
   methods: {
-    closeForm: function () {
+    closeForm: function (link) {
```

Comment:
try use es6 arrow function

Reply:
I updated functions in the method blocks to arrow functions